docker-compose.yml

```diff
@@ -9,6 +9,7 @@ web:
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+  command: bundle exec unicorn -l 0.0.0.0:3000
```

Comment:
I think it's really strange that it'd spin up Webrick instead of Unicorn if you explicitly specify a Unicorn server in the Dockerfile command. Kinda sniffs like it's using a cached image instead of the one specified in the Dockerfile. If that's the case, CircleCI would build and test an image built from the Dockerfile and then that would get deployed, but it could still be that the application on EB would break (since CircleCI does nothing like trying to access the server).